README.md

```diff
@@ -452,6 +491,18 @@ The `session` argument should be a session if found, otherwise `null` or
 `undefined` if the session was not found (and there was no error). A special
 case is made when `error.code === 'ENOENT'` to act like `callback(null, null)`.
 
+### store.load(sid, callback)
+
+**Required** (available from `express-session`)
+
+This required method is used to load a `Session` instance via the given session ID (`sid`). The `callback` should be called as `callback(error, session)`. This method calls `store.createSession` internally.
```

Comment:
Can you clarify what "This method calls store.createSession internally" means in the docs? Reading the docs, it sounds like I can either make it, or using the one from the module. The rest of the doc is about writing it myself, and so does that mean I have to call that method internally if I write it myself?